zipline/assets/asset_writer.py

```diff
@@ -165,6 +165,62 @@ def _generate_output_dataframe(data_subset, defaults):
     return output
 
 
+# Define a version number for the database generated by these writers
+# Increment this version number any time a breaking change is made to the
+# schema and readers of the database
+ASSET_DB_VERSION = 1
```

Comment:
I have a slight preference for `ASSETS_DB_VERSION`, but am happy to go either way. If you stick with `ASSET` you should update the following line: https://github.com/quantopian/zipline/pull/815/files#diff-33cdc2260291626609681cc61297818aR192